autocomplete/zsh/_heroku

```diff
@@ -10,6 +10,10 @@ _command_id () {
   _arguments "1:commands:($(grep -oe '^[a-zA-Z0-9:_-]\+' $COMMANDS_PATH))"
 }
 
+_apps () {
+  compadd "$@" - $(heroku autocomplete:values --cmd ${words[2]} --flag '--app')
```

Comment:
This was set up so when we don't know what the value is (i.e. when its more dynamic, not hardcoded) we can handle `-` or `--`